app/controllers/policies_controller.rb

```diff
@@ -1,62 +1,58 @@
 class PoliciesController < ApplicationController
   before_filter :clean_blank_parameters, only: [:create, :update]
-  expose(:policy, attributes: :policy_params)
-  expose(:policies) { Policy.includes(:parent_policies).order(:name) }
 
-  def index; end
+  def index
+    @policies = Policy.includes(:parent_policies).order(:name)
+  end
 
   def new
-    policy.sub_policy = true if params[:sub_policy]
+    @policy = PolicyForm.new_with_defaults
+    @policy.sub_policy = true if params[:sub_policy]
   end
 
   def create
-    if policy.save
-      Publisher.new(policy).publish!
-      flash[:success] = "Successfully created a policy"
+    policy_form = PolicyForm.from_form(policy_params)
```

Comment:
Not sure there's much benefit to the indirection provided by `from_form`. Could just call `new` here and it would make no odds.